mxcube3/__init__.py

```diff
@@ -110,6 +111,7 @@ def complete_initialization(app):
         app.empty_queue = pickle.dumps(hwr.getHardwareObject(cmdline_options.queue_model))
         app.sample_changer = app.beamline.getObjectByRole("sample_changer")
         app.rest_lims = app.beamline.getObjectByRole("lims_rest_client")
+        app.queue = qutils.new_queue()
```

Comment:
What is the difference between app.empty_queue and qutils.new_queue? Why would we need both?